lib/elixir_analyzer/quote_util.ex

```diff
@@ -68,15 +68,6 @@ defmodule ElixirAnalyzer.QuoteUtil do
     end)
   end
 
-  @doc """
-  Performs a depth-first, pre-order traversal of quoted expressions.
-  With depth provided to a function
-  """
-  @spec prewalk(Macro.t(), (Macro.t(), non_neg_integer -> Macro.t())) :: Macro.t()
-  def prewalk(ast, fun) when is_function(fun, 2) do
```

Comment:
I was a bit hesitant to get rid of this. It seems like it could be useful, but it's not actually used anywhere. I'm assuming at this point we won't need to add another complex analysis tool.

Reply:
No mercy for dead code! Even if it looks potentially useful, we will be able to recreate it when we need it. Making the code base easier to understand by removing trash is the better option every time, IMO.